app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadAdapter.kt

```diff
@@ -439,6 +444,33 @@ class ThreadAdapter(
         }
     }
 
+    private fun WebView.setupLinkContextualMenu(onClicked: (String) -> Unit) {
+        setOnLongClickListener {
+            val result = hitTestResult
+
+            when (result.type) {
+                WebView.HitTestResult.SRC_IMAGE_ANCHOR_TYPE,
+                WebView.HitTestResult.SRC_ANCHOR_TYPE -> {
+                    getLinkFromResult(result)?.let { url -> onClicked(url) }
```

Comment:
```suggestion
                    getLinkFromResult(result)?.let(onClicked)
```
This needs to be checked, I'm not 100% sure it's the right way to write this.